corehq/apps/registration/views.py

```diff
@@ -89,7 +89,7 @@ def register_user(request):
 
         context.update({
             'form': form,
-            'legacy_password': getattr(settings, 'ENABLE_DRACONIAN_SECURITY_FEATURES', False),
+            'show_password_feedback': not getattr(settings, 'ENABLE_DRACONIAN_SECURITY_FEATURES', False),
```

Comment:
Are you sure that's correct? I think you want to show the feedback if `ENABLE_DRACONIAN_SECURITY_FEATURES` is True.